www/reza02/build/vitkov.html

```diff
@@ -0,0 +1,184 @@
+<!DOCTYPE html>
```

Comment:
vetsina souboru neni ani naformatovany ani validni dle validatoru, tak na to bacha

Reply:
Jo jo, jasný, validaci jsem zatím neřešil (to udělám na konec) a formátování upravím (tohle je výsledek buildu.